### PR TITLE
Add yast tftp-server test

### PR DESCRIPTION
**Pull request by skliu**
- Related ticket: https://progress.opensuse.org/issues/49322
- Needles: none
- Verification run: 
SLE15: http://10.67.20.133/tests/307
SLE12: http://10.67.20.133/tests/308

### CODE REVIEW

File: tests/yast2_cmd/yast_tftp_server.pm [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: this test checks that YaST's tftp-server module are
+#          setup, enabled and disabled correctly in cmd mode.
+# Maintainer: Shukui Liu <skliu@suse.com>
+
+package yast_tftp_server;
```
Comment (dzedro) [inline tests/yast2_cmd/yast_tftp_server.pm]: `package yast_tftp_server;` is not needed, you will not use anything from here anywhere else

Comment (skliu) [reply tests/yast2_cmd/yast_tftp_server.pm]: `package yast_tftp_server;` is deleted.

**Comment by skliu** [comment]
> After download "tmp.txt", add a step to check "hello world" exist in this file.

done.

**Comment by dzedro** [comment]
I couldn't find this test scheduled on another SLE or openSUSE, but maybe it will not work somewhere properly

**Pull request merged**